src/dtclient/common_tenant_info_test.go

```diff
@@ -0,0 +1,51 @@
+package dtclient
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func tenantMalformedJson(url string) http.HandlerFunc {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		if request.URL.Path == url {
+			_, _ = writer.Write([]byte("this is not json"))
```

Comment:
What's the point of this syntax?

```suggestion
			writer.Write([]byte("this is not json"))
```

Reply:
I think this is needed by the linter. 
It makes it explicit that you don't care about the error, which is preferable to just not caring at all